aspnetcore/security/authorization/roles.md

```diff
@@ -138,8 +138,8 @@ If multiple attributes are applied at the controller and action levels, ***all**
 
 In the preceding `ControlAllPanelController` controller:
 
-* Members of the `Administrator` role or the `PowerUser` role can access the controller and the `SetTime` action.
-* Only members of the `Administrator` role can access the `ShutDown` action.
+* Members of the `Administrator` role can access the controller and the `SetTime` action.
+* Only members of the `Administrator` **and** the `PowerUser` role can access the `ShutDown` action.
```

Comment:
I added a screenshot of the current docs page (https://learn.microsoft.com/en-us/aspnet/core/security/authorization/roles?view=aspnetcore-5.0#adding-role-checks-1), because this might be easier to review.

![image](https://github.com/user-attachments/assets/adb05185-9f6e-4884-9f28-9ff1e5c423f4)